context/app/static/js/components/Routes/Route/Route.tsx

```diff
@@ -29,15 +29,15 @@ function Route({ children, disableWidthConstraint = false }: PropsWithChildren<{
   const shouldShowBoundaries = !disableWidthConstraint && isDesktop;
 
   return (
-    <Suspense fallback={<RouteLoader />}>
-      <Stack direction="row" width="100%">
+    <Stack direction="row" width="100%" flexGrow={1}>
```

Comment:
What necessitated this change?

Reply:
This was added to make the footer stick to the bottom of the screen.

Currently, if the viewport is taller than the page content, the footer appears halfway through the page:
![image](https://github.com/user-attachments/assets/446a9efa-f0ac-485f-b360-de8d9c8a03d1)

With this change, the footer is pushed to the bottom of the window in these scenarios: 
![image](https://github.com/user-attachments/assets/3cedbb3f-0c2c-4c3e-a053-301e569278a7)

I figured this would be a worthwhile change to make at this stage since the collapsible detail page sections make this a much more common scenario, and confirmed with @tsliaw prior to proceeding.